test/extended/util/db_image_helpers.go

```diff
@@ -34,6 +34,12 @@ type MySQL struct {
 	MasterPodName string
 }
 
+// PostgreSQL is a PostgreSQL helper for executing commands
+type PostgreSQL struct {
```

Comment:
nm, I see why you created two structs, though you could still handle the fields jointly via composition.  but that's probably overkill.